slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -216,6 +226,18 @@ ::grpc::Status SLAMServiceImpl::GetJpegMap(const GetMapRequest *request,
 
 ::grpc::Status SLAMServiceImpl::GetCurrentPointCloudMap(
     const GetMapRequest *request, GetMapResponse *response) {
+    // if we have a map for localization, use it
```

Comment:
[nit] for consistency with rest of comments

```suggestion
    // If we have a map for localization, use it
```